ui/src/app/pages/analysis/runtime-panel.tsx

```diff
@@ -157,27 +150,72 @@ export const RuntimePanel = withCurrentWorkspace()(({workspace}) => {
                     closeOnClick
                     content={
                       <React.Fragment>
-                        <MenuItem style={styles.presetMenuItem}>General purpose analysis</MenuItem>
-                        <MenuItem style={styles.presetMenuItem}>Genomics analysis</MenuItem>
+                        {
+                          fp.filter(['displayName', 'General Analysis'], runtimePresets)
```

Comment:
I didn't know filter could be used this way where it can be provided with a key/value pair. Is there another lodash function that does the same thing? I was confused because I'm very used to the filter API of .filter[collection, predicate]

Reply:
There is not another lodash function that does the same thing.

Reply:
If you want to keep everything in lodash/fp and get the index you could do something like (pseudo-codish):
```javascript
fp.flow(
  fp.filter(['displayName', 'General Analysis']),
  fp.toPairs,
  fp.map(([index, preset]) => {...})
)(runtimePresets)
```

Reply:
Tried that before and it didn't work, works now; probably I just didn't tuple up the arguments before